### DISCUSSION TITLE
Jellyfin 10.10.0版本下似乎无法自动刮削

### DISCUSSION THREAD

**Discussion by AresSun112** [metatube-community/jellyfin-plugin-metatube#420] [Q&A] [answered]
如题，Jellyfin 10.10.0版本下无法自动刮削，手动刮削也只能拉取封面图，但是无法生成标题、演员、简介、标签等等，是我Jellyfin版本的问题吗？还是我前后端有什么地方配置错了？此前在Jellyfin 10.8.13下是正常的。

**Comment by Topbcy** [comment]
我也遇到了，已經退回10.9.11

**Reply by AresSun112** [reply]
看来我不是孤例，那我就放心了，坐等后期更新修复，10.10.0用着蛮好的，此前很多bug得到了修复

**Reply by xjasonlyu** [reply]
好奇怪😬😬😬

**Reply by Topbcy** [reply]
> 好奇怪😬😬😬

我發現的狀況有些可以刮削，有些不行，不過大部分都是不行

**Reply by xjasonlyu** [reply]
哪些情况下不行？比如某个番号，我到时候看看

**Reply by Topbcy** [reply]
> 看来我不是孤例，那我就放心了，坐等后期更新修复，10.10.0用着蛮好的，此前很多bug得到了修复

哥你提供一下~~

**Reply by AresSun112** [reply]
> > 看来我不是孤例，那我就放心了，坐等后期更新修复，10.10.0用着蛮好的，此前很多bug得到了修复
> 
> 哥你提供一下~~

啊？我是都不行，得手动

**Reply by Topbcy** [reply]
> 哪些情况下不行？比如某个番号，我到时候看看

我猜我有些可以是因為有metatube.db的關係，經反覆驗證後都不行。

**Comment by xjasonlyu** [comment, marked as answer]
Fixed in #422

**Reply by AresSun112** [reply]
> Fixed in #422

谢谢，已解决
